easybuild/easyblocks/c/cuda.py

```diff
@@ -234,6 +234,17 @@ def create_wrapper(wrapper_name, wrapper_comp):
             raise EasyBuildError("Unable to find 'ldconfig' in $PATH (%s), nor in any of %s", path, sbin_dirs)
 
         stubs_dir = os.path.join(self.installdir, 'lib64', 'stubs')
+
```

Comment:
Isn't this in the wrong location now? `stubs_dir` is empty at this point so for loop is pointless (change made in https://github.com/easybuilders/easybuild-easyblocks/pull/2793/commits/cd3f0bbd9ce6b104a69afe4d953d9d2a874d4fa9)